ansible/roles/dev/tasks/main.yml

```diff
@@ -135,8 +135,24 @@
       - app
       - common
       - exceptions
+      - plugin
       - tasking
 
+# We don't currently have a common method for installing plugins for development. In Pulp 2, it was
+# "run pulp-dev.py -I", so maybe that's what we'll do for pulp 3 as well. For now, since only the
+# file plugin works with "python setup.py develop", we'll just try that
+# exists, installing plugins to the platform virtualenv
+- name: Install Plugin packages into Platform Virtualenv
+  command: "{{ pulp_venv_dir }}/pulp/bin/python setup.py develop"
```

Comment:
The pulp packages don't have to be installed system-wide, that's just a timesaving step to effectively install them into every virtualenv, so we don't have to loop over every virtualenv and then loop over all the pulp.* packages and install them.

Since the plugins can't even be imported, let along function, without platform, and since platform is effectively nonfunctional if no plugins are installed, it (sorta?) makes sense to get all the plugins installed there, which still allowing for working on a plugin in isolation by "workon"-ing that plugin's virtualenv.

Though, comically, the pulp_file package isn't yet installed into the pulp_file virtualenv. Probably it should be, and I can add that in here as well if you think it's worth it.